tests/testthat/test-trailing_blank_lines_linter.R

```diff
@@ -7,7 +7,7 @@ test_that("trailing_blank_lines_linter doesn't block allowed usages", {
   expect_lint("blah <- 1\nblah\n \n blah", NULL, linter)
 
   tmp <- withr::local_tempfile(lines = "lm(y ~ x)")
-  expect_lint(content = tmp, file = tmp, NULL, linter)
+  expect_lint(file = tmp, checks = NULL, linter)
```

Comment:
how does this work? Won't `lintr` wind up positionally as the `content=` argument?

(please apply the solution elsewhere in the file as well)

Reply:
Yeah, puzzling indeed.

At any rate, I have done this clean-up for this file and also for `test-knitr_format.R` file.

Reply:
Thanks, I think we have this part fixed. Just a few last issues with temp files to address.